packages/edge/README.md

```diff
@@ -31,6 +31,65 @@ Install the Adobe Experience Platform Edge Network extension in your mobile prop
 Then follow the same document for registering the Edge extension with the Mobile Core.
 Note that initializing the SDK should be done in native code, additional documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
 
+
+Initializing Example:
+
+iOS
+```objc
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+...
+@implementation AppDelegate
+
+// AppDelegate.m
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [AEPMobileCore registerExtensions:@[AEPMobileEdgeIdentity.class, 
+                                        AEPMobileEdge.class] completion:^{
+     ...	
+
+   }];
+   [AEPMobileCore configureWithAppId:@"yourAppID"];   
+ } 
+
+@end
+```
+
+Android
+```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Edge;
+  
+...
+import android.app.Application;
+...
+public class MainApplication extends Application implements ReactApplication {
+  ...
+  @Override
+  public void on Create(){
+    super.onCreate();
+    ...
+    MobileCore.setApplication(this);
+    MobileCore.setLogLevel(LoggingMode.DEBUG);
+    MobileCore.setWrapperType(WrapperType.REACT_NATIVE);
+    MobileCore.configureWithAppID("yourAppID");
+
+    try {
+      Edge.registerExtension();
+      com.adobe.marketing.mobile.edge.identity.Identity.registerExtension();
+      MobileCore.start(new AdobeCallback() {
+        @Override
+        public void call(Object o) {
+        
+        }});
+    } catch (InvalidInitException e) {
```

Comment:
can we remove the try catch? I believe it is not required for the Edge extensions